Makefile

```diff
@@ -0,0 +1,57 @@
+SHELL:=bash
```

Comment:
Has this file been copied from the vitals work? Theres a bunch of comments etc that contain 'vitals' as well as the `delete-test-dir` step